Modules/Filtering/ImageGrid/include/itkResampleImageFilter.hxx

```diff
@@ -61,7 +61,7 @@ ResampleImageFilter<TInputImage, TOutputImage, TInterpolatorPrecisionType, TTran
 
   // "Transform" required ( not numbered )
   Self::AddRequiredInputName("Transform");
-  Self::SetTransform(IdentityTransform<TTransformPrecisionType, ImageDimension>::New());
+  // Self::SetTransform(IdentityTransform<TTransformPrecisionType, ImageDimension>::New());
```

Comment:
I left it as a comment because I wasn't sure how to deal with this. We can set set it to identity transform if input and output dimension are same.. or we could create a new rectangular identity transform.

Reply:
> set it to identity transform if input and output dimension are same

This sounds reasonable and backwards compatible.

Reply:
will fix this is next commit.